plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableProperties.java

```diff
@@ -153,6 +155,8 @@ public HiveTableProperties(
                 stringProperty(CSV_SEPARATOR, "CSV separator character", null, false),
                 stringProperty(CSV_QUOTE, "CSV quote character", null, false),
                 stringProperty(CSV_ESCAPE, "CSV escape character", null, false),
+                stringProperty(REGEX_PATTERN, "REGEX pattern", null, false),
```

Comment:
We could do the validation here
```java
stringProperty(REGEX_PATTERN, "REGEX pattern", null, Pattern::compile, false),
```
Or to avoid the warning
```java
@SuppressWarnings("ResultOfMethodCallIgnored") 
Consumer<String> regexValidator = Pattern::compile;

stringProperty(REGEX_PATTERN, "REGEX pattern", null, regexValidator, false),

Reply:
To match the other table properties we decided to leave this as is, but it is something we can reconsider in the future.